cmd/main.go

```diff
@@ -113,21 +116,15 @@ func main() {
 		// TODO(user): If CertDir, CertName, and KeyName are not specified, controller-runtime will automatically
 		// generate self-signed certificates for the metrics server. While convenient for development and testing,
 		// this setup is not recommended for production.
-
-		// TODO(user): If cert-manager is enabled in config/default/kustomization.yaml,
-		// you can uncomment the following lines to use the certificate managed by cert-manager.
```

Comment:
Leaving this in wouldn't hurt for whoever ends up making the cut-over to requiring cert-manager.